rct229/rulesets/ashrae9012019/section19/section19rule14.py

```diff
@@ -0,0 +1,273 @@
+)
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_zone_supply_return_exhaust_relief_terminal_fan_power_dict import (
+    get_zone_supply_return_exhaust_relief_terminal_fan_power_dict,
+)
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all, find_one
+from rct229.utils.pint_utils import ZERO
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_1,
+    HVAC_SYS.SYS_2,
+    HVAC_SYS.SYS_3,
+    HVAC_SYS.SYS_4,
+    HVAC_SYS.SYS_5,
+    HVAC_SYS.SYS_6,
+    HVAC_SYS.SYS_7,
+    HVAC_SYS.SYS_8,
+    HVAC_SYS.SYS_11_1,
+    HVAC_SYS.SYS_12,
+    HVAC_SYS.SYS_13,
+]
+
+
+class Section19Rule14(RuleDefinitionListIndexedBase):
+    """Rule 14 of ASHRAE 90.1-2019 Appendix G Section 19 (HVAC - General)"""
+
+    def __init__(self):
+        super(Section19Rule14, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, True),
+            each_rule=Section19Rule14.HVACRule(),
+            index_rmr="baseline",
+            id="19-14",
+            description="For baseline system types 1-8 and 11-13, if return or relief fans are specified in the proposed design, the baseline building design shall also be modeled with fans serving the same functions and sized for the baseline system supply fan air quantity less the minimum outdoor air, or 90% of the supply fan air quantity, whichever is larger.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1.2.8.1 Excluding Exceptions 1 and 2",
+            is_primary_rule=True,
+            rmr_context="ruleset_model_instances/0",
+            list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    def create_data(self, context, data):
+        rmi_b = context.baseline
+        rmi_p = context.proposed
+
+        dict_of_zones_and_terminal_units_served_by_hvac_sys_b = (
+            get_dict_of_zones_and_terminal_units_served_by_hvac_sys(rmi_b)
+        )
+        zone_supply_return_exhaust_relief_terminal_fan_power_dict = (
+            get_zone_supply_return_exhaust_relief_terminal_fan_power_dict(rmi_p)
+        )
+
+        hvac_info = {}
+        for hvac_b in find_all(
+            "$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+            rmi_b,
+        ):
+            hvac_id_b = hvac_b["id"]
+            hvac_info[hvac_id_b] = {}
+
+            fan_system_info_b = (
+                get_zone_supply_return_exhaust_relief_terminal_fan_power_dict(
+                    getattr_(hvac_b, "HVAC", "fan_system")
+                )
+            )
+
+            hvac_info[hvac_id_b]["more_than_one_supply_or_return_fan"] = (
+                False
+                if (
+                    fan_system_info_b["supply_fans_qty"] == 1
+                    and fan_system_info_b["return_fans_qty"] == 1
+                )
+                else True
+            )
```

Comment:
If the nested class is HVAC, I would prefer to get this information inside the nested class.

Reply:
Addressed.